install.ps1

```diff
@@ -0,0 +1,80 @@
+# Installs KoreBuild into a repo, overwriting all previous build scripts
+# This script is downloaded and executed without any of the corresponding scripts in this repo, so don't assume anything in this repo is available!
+
+param(
+    [string]$KoreBuildRepo,
+    [string]$KoreBuildBranch = "anurse/msbuild") # TEMPORARY until this is merged to dev
+
+# Prerequisites
+if(!(Get-Command git -ErrorAction SilentlyContinue)) {
+    throw "KoreBuild requires Git"
```

Comment:
Here, we're talking about **installation**. CIs shouldn't be installing KoreBuild into repos, only using it.

We do use `git` in the build script itself to ensure KoreBuild is always up-to-date. I think the benefits of that (never having to delete `.build` to force an update) outweigh the costs of installing git on a couple of machines, but I'm certainly open to discussion.